Core/include/Acts/Utilities/BinAdjustment.hpp

```diff
@@ -142,6 +192,12 @@ BinUtility adjustBinUtility(const BinUtility& bu, const Surface& surface) {
     // Return specific adjustment
     return adjustBinUtility(bu, *rBounds, surface.transform(GeometryContext()));
   }
+  else if (surface.type() == Surface::Plane && surface.bounds().type() == SurfaceBounds::eRectangle) {
```

Comment:
Thanks a lot for your comments, Corentin!

I added a Unit test that returned no errors. I also split up the if as you suggested.

I also ran the CI/check_format script, but I think the check failed again. I am running acts in one of the acts docker-containers which did not contain clang-format. So I ran clang-format outside the docker-container environment with "./CI/check_format ." and it seemed to be happy, this is what it returned: 

```
LLVM (http://llvm.org/):
  LLVM version 3.4.2
  Optimized build.
  Built May 10 2018 (10:48:27).
  Default target: x86_64-redhat-linux-gnu
  Host CPU: x86-64
```
Maybe I did something wrong, or maybe it fails because I have executed it outside the docker but usually run acts within?

Reply:
Hey @mesmerisa, unfortunately, you'll need to use clang-format version 10 to do this, since the format result differs by version.

You have 4 options:
1. Use our docker based alternativ `CI/check_format_local`, if you have docker installed
2. Build LLVM10 including clang-format locally
3. Use a statically linked binary of clang-format (ping me in Mattermost if you want this)
4. The failing format CI job provides a build artifact with the reformatted files, which you can copy over your local version and commit that.

Reply:
Thanks a lot for the input @paulgessinger! 
I have docker installed, I tried option 1 first, but I get `/bin/sh: 0: Can't open CI/check_format` after it downloaded the image. What would you recommend?

Reply:
Which directory are you running the script from?

Reply:
I am running it directly from acts via `./CI/check_format_local .`.

Reply:
OK then maybe that's broken at the moment. Sorry.

Reply:
Ok, I tried on my ubuntu laptop again and there it worked! 
(Before I tried on centos7).

Reply:
> Hey @mesmerisa, unfortunately, you'll need to use clang-format version 10 to do this, since the format result differs by version.
> 
> You have 4 options:
> 
> 1. Use our docker based alternative `CI/check_format_local`, if you have docker installed
> 2. Build LLVM10 including clang-format locally
> 3. Use a statically linked binary of clang-format (ping me in Mattermost if you want this)
> 4. The failing format CI job provides a build artifact with the reformatted files, which you can copy over your local version and commit that.

Actually there are also binaries you can pull for `clang-format`, @paulgessinger has pointed me to those ones, and I have changed to this.